.github/workflows/nb_server_jupyter_scipy_docker_publish.yaml

```diff
@@ -0,0 +1,64 @@
+name: Build & Publish Notebook Server Docker images
```

Comment:
@NickLoukas is there a reason there is no "Jupyter SciPy" in the name here? I mean, in the titles of all other nb server docker publish workflows, the target architecture is mentioned.